install/pre-install-payload/scripts/reqs-deploy.sh

```diff
@@ -87,6 +110,24 @@ function uninstall_containerd_artefacts() {
 	fi
 }
 
+function uninstall_nydus_snapshotter_artefacts() {
+	remove_nydus_snapshotter_from_containerd
+
+	host_systemctl disable nydus-snapshotter.service
```

Comment:
Shouldn't stop the snapshotter service before? or passing `---now` to systemctl's disable should do the trick too.

Reply:
@fidencio finished my review. This is the only question preventing me of approving it.

Reply:
> Shouldn't stop the snapshotter service before? or passing ---now to systemctl's disable should do the trick too.

Very nice catch!